js/src/sdk/models/triggers.ts

```diff
@@ -4,12 +4,20 @@ import { BackendClient } from "./backendClient";
 
 import apiClient from "../client/client";
 
-import { ListTriggersData } from "../client";
+import { z } from "zod";
 import { CEG } from "../utils/error";
 import { TELEMETRY_LOGGER } from "../utils/telemetry";
 import { TELEMETRY_EVENTS } from "../utils/telemetry/events";
 
-type RequiredQuery = ListTriggersData["query"];
+const ZTriggerQuery = z.object({
+  triggerIds: z.array(z.string()).optional(),
```

Comment:
Consider adding input validation for empty strings in the array elements and maximum array length limits to prevent potential API performance issues. For example:
```typescript
triggerIds: z.array(z.string().min(1)).max(100).optional(),
```